src/VisualStudio/Core/Def/Progression/IProgressionLanguageService.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Progression;
+
+internal interface IProgressionLanguageService : ILanguageService
+{
+    IEnumerable<SyntaxNode> GetTopLevelNodesFromDocument(SyntaxNode root, CancellationToken cancellationToken);
```

Comment:
I imagine this method isn't being used if you wanted to delete this to keep it clearer what's left.

Reply:
good call.  done.